content/content-tool/tool/src/webapp/vm/content/sakai_resources_restore.vm

```diff
@@ -149,6 +149,15 @@
 			submitform(id);
 		}
 		
+		var submitting = 0; 
+		function submitThisOnce(id, flow)
+		{
+			if (submitting == 0 ) {
+				submitting = 1;
+   				submitThis(id, flow);
+   			}		
+		}
+		
```

Comment:
I know I am being picky, but making var submitting a true/false value would be more concise.

It probably makes sense to disable the button as well after the submit, kind of letting the user know you can't click it again?